prow/plugins/approve/approve.go

```diff
@@ -585,13 +585,24 @@ func addApprovers(approversHandler *approvers.Approvers, approveComments []*comm
 // optionsForRepo gets the plugins.Approve struct that is applicable to the indicated repo.
 func optionsForRepo(config *plugins.Configuration, org, repo string) *plugins.Approve {
 	fullName := fmt.Sprintf("%s/%s", org, repo)
+
+	// First search for repo config
+	for i := range config.Approve {
```

Comment:
I wonder why these two fields don't use `for i, a := range config.Approve { if a.Repos { return &a } }`

Reply:
Yeah, we don't *really* care about the key here, only the value.. so I changed that. Good idea.